victoriametrics-operator/README.md

```diff
@@ -0,0 +1,11 @@
+victoriametrics-operator
+========================
+
+Kubernetes operator for Victoria Metrics
+
+- [VictoriaMetrics operator)](https://github.com/VictoriaMetrics/operator)
```

Comment:
The following are trivial points; fix them if there are also important ones.

Remove ")" from the link phrase.

Use "VictoriaMetrics" w/o a space in l. 4 too.
This style would be preferable because it is used almost everywhere - except for the top of the official web page!